bibtexparser/middlewares/names.py

```diff
@@ -175,15 +178,22 @@ class MergeNameParts(_NameTransformerMiddleware):
     The merging style indicates whether:
     - the merging is done without commas in first-name-first order ("first"), or
     - the merging is done with commas in last-name-first order ("last").
+
+    :param style: Merge style (see above).
+    :sep_first: Separator between first name parts.
+    :param allow_inplace_modification: See corresponding property parent class.
+    :param name_fields: The fields that contain names.
     """
 
     def __init__(
         self,
         style: Literal["last", "first"] = "last",
+        sep_first: str = " ",
```

Comment:
Have you considered making this a setter to keep the constructor less verbose? I currently imagine (but please tell me what I miss) that barely anyone would use this, thus adding four more constructor params (one for each separator) seems a bit heavy. 

What are your thoughts on that?